xbmc/rendering/dx/DeviceResources.cpp

```diff
@@ -1484,3 +1484,20 @@ bool DX::DeviceResources::SetMultithreadProtected(bool enabled) const
 
   return (wasEnabled == TRUE ? true : false);
 }
+
+bool DX::DeviceResources::IsGCNOrOlder() const
+{
+  if (CSysInfo::GetWindowsDeviceFamily() == CSysInfo::Xbox)
+    return false;
+
+  if (GetAdapterDesc().VendorId != PCIV_AMD)
+    return false;
+
+  VideoDriverInfo driver = DX::DeviceResources::Get()->GetVideoDriverVersion();
```

Comment:
```suggestion
  const VideoDriverInfo driver = GetVideoDriverVersion();
```

Reply:
`GetVideoDriverVersion()`is a member method of `DeviceResources`. 

Why you need `DX::DeviceResources::Get()->` to use it internally?

Reply:
It was initially in a different class. Replaced with a direct call to `CWIN32Util::GetVideoDriverInfo()` to avoid double call to GetAdapter and not necessary logging.